content/community/tools-jsx.md

```diff
@@ -1,53 +1,57 @@
+* **[nwb](https://github.com/insin/nwb)**: عُدّة لـReact و Preact و تطيبقات Inferno و مكتبات React و وحدات npm أخرى للويب وبدون إعدادات (إلى أن أنت تحتاجها).
+* **[Neutrino](https://neutrino.js.org/)**: إنشاء وبناء تطبيقات JavaScript متقدمة بصفر إعدادات أولية. يجمع Neutrino بين قوة webpack مع بساطة المسبقة.
+* **[ESLint](https://eslint.org/):** دقق JavaScript قابل للإضافة يدعم اصلا صيغة JSX. تأكد من تحميل [eslint-plugin-react](https://npmjs.com/package/eslint-plugin-react) لقواعد React-specific.
+* **[Structor](https://www.npmjs.com/package/structor):** هذه الأداة  لواجهة لمسخدم لبناء تطبيقات الويب بـNode.js مع React UI. Structor استبدلت بـReact UI Builder المهملة. شاهد [الفيديو التعليمي لـStructor](https://youtu.be/z96xYa51EWI?list=PLAcaUOtEwjoR_U6eE2HQEXwkefeVESix1).
+* **[react-jsx](https://github.com/bigpipe/react-jsx):** إعادة الصياغة وإستعمال JSX كنموذج مستقل يمكن إستعماله في جهة الخادم أو المستخدم!
+* **[cjsx-codemod](https://github.com/jsdf/cjsx-codemod):** كتابة شفرة JSX من خلال Coffeescript!
+* **[ReactScript](https://github.com/1j01/react-script):** كتابة شفرة React من خلال Coffeescript وبدون JSX!
 * **[jsxhint](https://npmjs.org/package/jsxhint):** [JSHint](http://jshint.com/) (linting) support. (JSX compilation doesn't affect line numbers so lint can also be run directly on the compiled JS.)
-* **[reactify](https://npmjs.org/package/reactify):** [Browserify](http://browserify.org/) transform.
+
+* **[reactify](https://npmjs.org/package/reactify):** تحويل[Browserify](http://browserify.org/).
 * **[Babel](https://babeljs.io/):** Standalone & [Browserify](http://browserify.org/) transform (formerly known as **6to5**).
-* **[node-jsx](https://npmjs.org/package/node-jsx):** Native [Node](https://nodejs.org/) support.
-* **[react-hot-loader](https://gaearon.github.io/react-hot-loader/):** Loader for [webpack](https://webpack.github.io/) that lets you edit JSX and have changes appear immediately in the browser without reloading the page.
-* **[jsx-loader](https://npmjs.org/package/jsx-loader):** Loader for [webpack](https://webpack.github.io/).
-* **[express-jsxtransform](https://www.npmjs.org/package/express-jsxtransform):** Middleware for [Express](https://www.npmjs.org/package/express).
+مستقل و تحويل [Browserify](http://browserify.org/) (معروف سابقا بإسم **6to5**).
+* **[node-jsx](https://npmjs.org/package/node-jsx):** [Node](https://nodejs.org/) الأصلي يدعه.
+* **[react-hot-loader](https://gaearon.github.io/react-hot-loader/):** مُحَمّل لـ[webpack](https://webpack.github.io/) يتيح لك تحرير JSX وتغيير دون إعادة تحميل صفحة المتصفح على الإطلاق.
+* **[jsx-loader](https://npmjs.org/package/jsx-loader):** مُحَمّل لـ[webpack](https://webpack.github.io/).
+* **[express-jsxtransform](https://www.npmjs.org/package/express-jsxtransform):** أدوات وسيطية لـ[Express](https://www.npmjs.org/package/express).
 * **[gradle-react-plugin](https://github.com/ehirsch/gradle-react-plugin):** Transform jsx sources during a gradle build.
```

Comment:
You may forget to remove this :)
```suggestion
* **[gradle-react-plugin](https://github.com/ehirsch/gradle-react-plugin):** 
```

Reply:
Done

Reply:
Please make sure to push the updates, it's not live on [your deployment](https://deploy-preview-157--ar-reactjs.netlify.com/community/jsx-integrations.html#build-tools)